microshift_troubleshooting/microshift-things-to-know.adoc

```diff
@@ -17,4 +17,8 @@ When such changes occur, some {microshift-short} components may stop functioning
 
 The threshold for clock changes is a time adjustment of greater than 10 seconds in either direction. Smaller drifts on regular time adjustments performed by the Network Time Protocol (NTP) service do not cause a restart.
 
-include::modules/microshift-certificate-lifetime.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/microshift-certificate-lifetime.adoc[leveloffset=+1]
+[role="_additional-resources"]
```

Comment:
```suggestion
include::modules/microshift-certificate-lifetime.adoc[leveloffset=+1]
+
[role="_additional-resources"]
```

Just incase some weird rendering issue occurs.

Reply:
@tmalove Need to remove the `+` in line 23. It is causing rendering issues. 

![image](https://github.com/user-attachments/assets/89417285-1f1f-493c-ab40-4b6e45e5ef6c)